mmv1/products/cloudsecuritycompliance/FrameworkDeployment.yaml

```diff
@@ -0,0 +1,305 @@
+                    required: true
+                  - name: parameterValue
+                    type: NestedObject
+                    description: Possible parameter value types.
+                    required: true
+                    properties:
+                      - name: boolValue
+                        type: Boolean
+                        description: Represents a boolean value.
+                      - name: numberValue
+                        type: Double
+                        description: Represents a double value.
+                      - name: stringListValue
+                        type: NestedObject
+                        description: A list of strings.
+                        properties:
+                          - name: values
+                            type: Array
+                            description: The strings in the list.
+                            required: true
+                            item_type:
+                              type: String
+                      - name: stringValue
+                        type: String
+                        description: Represents a string value.
+        - name: enforcementMode
+          type: String
+          description: |-
+            Enforcement mode for the framework deployment.
+            Possible values:
+            PREVENTIVE
+            DETECTIVE
+            AUDIT
+          required: true
+  - name: computedTargetResource
+    type: String
+    description: |-
+      The resource on which the Framework is deployed based on the provided
+      TargetResourceConfig in the following format:
+      organizations/{organization}, folders/{folder} or projects/{project}
+    output: true
+  - name: createTime
+    type: String
+    description: The time at which the resource was created.
+    output: true
+  - name: deploymentState
+    type: String
+    description: |-
+      The deployment state of the framework.
+      Possible values:
+      DEPLOYMENT_STATE_VALIDATING
+      DEPLOYMENT_STATE_CREATING
+      DEPLOYMENT_STATE_DELETING
+      DEPLOYMENT_STATE_FAILED
+      DEPLOYMENT_STATE_READY
+      DEPLOYMENT_STATE_PARTIALLY_DEPLOYED
+      DEPLOYMENT_STATE_PARTIALLY_DELETED
+    output: true
+  - name: description
+    type: String
+    description: User provided description of the Framework deployment
+  - name: etag
+    type: String
+    description: |-
+      To prevent concurrent updates from overwriting each other, always provide
+      the `etag` when you update a FrameworkDeployment. You can also
+      provide the `etag` when you delete a FrameworkDeployment, to help
+      ensure that you're deleting the intended version of the
+      FrameworkDeployment.
+    output: true
+  - name: framework
+    type: NestedObject
+    description: FrameworkReference contains the reference of a framework.
+    required: true
+    properties:
+      - name: framework
+        type: String
+        description: |-
```

Comment:
Just want to flag - the API doc is not correct for this field at https://docs.cloud.google.com/security-command-center/docs/reference/cloudsecuritycompliance/rest/v1/FrameworkReference